phpmp3.php

```diff
@@ -520,15 +361,13 @@ public function save($path)
 	}
 	
 	//join various MP3s
-    public function multiJoin($newpath, $array)
-    {
-        foreach ($array as $path)
-        {
-            $mp3 = new PHPMP3($path);
-            $mp3->striptags();
-            $mp3_1 = new PHPMP3($newpath);
-            $mp3->mergeBehind($mp3_1);
-            $mp3->save($newpath);
+    public function multiJoin($newpath, $array) {
+        foreach ($array as $path) {
+            $mp3 = new PHPMP3 ($path);
+            $mp3->striptags ();
+            $mp3_1 = new PHPMP3 ($newpath);
+            $mp3->mergeBehind ($mp3_1);
+            $mp3->save ($newpath);
```

Comment:
For future pull requests, please try not to change the coding style.

I'm going to update the code to use psr-2 coding style.